raster/r.colors.out/raster_main.c

```diff
@@ -54,10 +59,25 @@ int main(int argc, char **argv)
     opt.file->description = _("If not given write to standard output");
     opt.file->required = NO;
 
+    opt.format = G_define_standard_option(G_OPT_F_FORMAT);
+    opt.format->guisection = _("Print");
+
     flag.p = G_define_flag();
     flag.p->key = 'p';
     flag.p->description = _("Output values as percentages");
 
+    flag.r = G_define_flag();
```

Comment:
@cwhite911 @wenzeslaus correct me if I am wrong, but I don't think we want these flags, all the formats should be available in the JSON output.

Reply:
@cwhite911 @wenzeslaus Just to confirm, would you like me to make this change so that all formats are included in the JSON output without the flags?

Reply:
@petrasovaa I was thinking it would be nice to specify what you want verses including all color formats since you generally only need one.

Reply:
Would an option be better than flags, e.g. color_format=rgb?
Example how to write and parameter with multiple options:
https://github.com/OSGeo/grass/blob/main/vector/v.to.rast/main.c#L77

Reply:
The key-value "options" are generally more readable than one letter flags. That's what we went for with format=json.

Reply:
Sure, adding the `color_format` option instead of a flag now.

Reply:
done, ptal...